Create decoct.md for PHP Math Functions

reganmarie opened the pull request:
<!--
👋 Hi, thanks for submitting a PR to Codecademy Docs! 💖
Please fill out all fields below to ensure your PR is reviewed quickly.

**IMPORTANT**

If you would like to receive credit for your contribution to an entry, make sure to have your Codecademy user profile linked to your GitHub account:

1. Go to your Codecademy dashboard.
2. Click on your profile image in the top-right and then choose "Profile".
3. Then click "Edit Profile".
4. In the "GitHub Username" field, add your username (without the @).
5. Click "Save Changes"

Of course, you can opt not to do this and be listed as an "Anonymous contributor", instead. :)
-->

### Description

- Closes #2623 
- Created decoct.md on docs/content/php/concepts/math-functions/terms/decoct/decoct.md
- Added Title, Description, Subjects, Tags, and Catalog Content
- Added description, syntax, example, and codebyte example
- Ran through 3 online plagiarism checkers to ensure 100% unique wording and description: passed with 100%
- Followed content and style guidelines -Checked for spelling and grammar errors
- Corrected for clarity and readability

### Type of Change

<!--- Please delete or cross off the bullet point(s) that are irrelevant to this PR: -->

- Adding a new entry

### Checklist

<!-- Please check ALL the boxes: -->

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

<!--
Having trouble with the PR checker? Here are some common issues and resolutions:

- verify_formatting is failing
  - run `yarn format path/to/markdown/file.md` or `yarn format:all` and commit the results
- verify_lint is failing
  - same as above
  - if verify_lint is still failing, running `yarn lint` locally should let you know what needs to be changed by hand
- test is failing
  - ensure any new markdown files have a `Title` and `Description` defined in their metadata
  - ensure any new markdown files only contain alphanumerics and dashes in their file names and have the same name as their parent directory
  - if that looks ok, running `yarn test` locally should let you know what the issue is
-->

dakshdeepHERE commented on content/php/concepts/math-functions/terms/decoct/decoct.md:
```diff
@@ -0,0 +1,54 @@
+---
+Title: 'decoct()'
```
```suggestion
Title: '.decoct()'
```

dakshdeepHERE commented on content/php/concepts/math-functions/terms/decoct/decoct.md:
```diff
@@ -0,0 +1,54 @@
+  - 'paths/computer-science'
+---
+
+The **`decoct()`** function takes a given decimal number and returns a string version of the corresponding octal value.
+
+An octal value is from the base-8 numerical system, which only uses eight digits: zero through seven.
+
+For 32-bit platforms, 4294967295 is the largest number that can be computed. For 64-bit platforms, 9223372036854775807 is the largest number that can be computed.
```
```suggestion
The **`.decoct()`** function takes a given decimal number and returns a `string` version of the corresponding octal value. An octal value is from the base-8 numerical system, which only uses eight digits: zero through seven. For 32-bit platforms, 4294967295 is the largest number that can be computed. For 64-bit platforms, 9223372036854775807 is the largest number that can be computed.
```

dakshdeepHERE commented on content/php/concepts/math-functions/terms/decoct/decoct.md:
```diff
@@ -0,0 +1,54 @@
+
+## Example
+
+This example uses the `decoct` function to convert a decimal to its octal value.
```
```suggestion
This example uses the `.decoct()` function to convert a decimal to its octal value.
```

dakshdeepHERE reviewed (changes requested):
Few changes from my side. Otherwise looks good.

reganmarie commented:
@dakshdeepHERE Made the suggested changes! 
- Updated `decoct()` to `.decoct()` in line 2, line 27, and line 43 to update per edit and for consistency
- Changed lines 15-19 to enclose string in `` and to take up one line versus being spaced out

dakshdeepHERE reviewed (approved):
@reganmarie this looks good for second review. 🚀

reganmarie commented:
Thank you @dakshdeepHERE and @KTom101! 😊

caupolicandiaz reviewed (approved):
@reganmarie @dakshdeepHERE LGTM
